src/FilamentMenuBuilderPlugin.php

```diff
@@ -143,20 +153,55 @@ public function showCustomTextPanel(bool $show = true): static
         return $this;
     }
 
-    public function addMenuFields(array | Closure $schema): static
+    public function addMenuFields(array|Closure $schema): static
     {
         $this->menuFields = $schema;
 
         return $this;
     }
 
-    public function addMenuItemFields(array | Closure $schema): static
+    public function addMenuItemFields(array|Closure $schema): static
     {
         $this->menuItemFields = $schema;
 
         return $this;
     }
 
+    public function navigationLabel(string|Closure $label = null): static
+    {
+        $this->navigationLabel = $label;
+
+        return $this;
+    }
+
+    public function navigationGroup(string|Closure $group = null): static
+    {
+        $this->navigationGroup = $group;
+
+        return $this;
+    }
+
+    public function navigationIcon(string|Closure $icon): static
+    {
+        $this->navigationIcon = $icon;
+
+        return $this;
+    }
+
+    public function navigationSort(int|Closure $order): static
+    {
+        $this->navigationSort = $order;
+
+        return $this;
+    }
+
+    public function navigationCountBadge(bool $show): static
```

Comment:
```suggestion
    public function navigationCountBadge(bool $show = true): static
```

Could you change the default value of `$show` to `true`?